app/src/main/java/org/mozilla/tv/firefox/components/locale/LocaleManager.java

```diff
@@ -283,14 +283,14 @@ public Locale getCurrentLocale(@NonNull Context context) {
         return currentLocale = Locales.parseLocaleCode(current);
     }
 
-    public boolean currentLanguageIsEnglish(@NonNull Context context) {
+    @NonNull
+    public String getCurrentLanguage(@NonNull Context context) {
```

Comment:
Why was this changed? It looks like it's only used in `PocketViewModel` to check for English anyway

Reply:
Tbh, I don't remember.  Maybe I thought it would be useful in other contexts as well?  I can change it back.